src/main/sensors/gyro.c

```diff
@@ -125,12 +136,16 @@ void gyroUpdate(void)
         return;
     }
 
-    if (gyroFIRTable) {
-        filterApplyFIR(gyroADC, gyroFIRState, gyroFIRTable);
-    }
-
     alignSensors(gyroADC, gyroADC, gyroAlign);
 
+    if (gyroLpfCutFreq) {
+        if (!gyroFilterStateIsSet) {
+            initGyroFilterCoefficients();
+        } else {
```

Comment:
The gyro is being laoded pretty much as one of the first things. There are not much options left unlike you want to do it in main.c or gyro_sync.c